inc/lift/EventLoop.h

```diff
@@ -102,11 +102,12 @@ class EventLoop {
     uv_async_t m_async {};
     /// libcurl requires a single timer to drive timeouts/wake-ups.
     uv_timer_t m_timeout_timer {};
+    uv_timer_t m_response_timer {};
     /// The libcurl multi handle for driving multiple easy handles at once.
     CURLM* m_cmh { curl_multi_init() };
 
     /// Pending requests are safely queued via this lock.
-    std::mutex m_pending_requests_lock {};
+    mutable std::mutex m_pending_requests_lock{};
```

Comment:
Minor: Suggest naming this "..._mutex" rather than "..._lock" (lock is a separate object/action) -- but maybe fine as-is since this isn't new

Reply:
Good point